cli/filelister.cpp

```diff
@@ -189,12 +189,16 @@ static void addFiles2(std::map<std::string, std::size_t> &files,
             if (!dir)
                 return;
 
-            dirent entry;
             dirent * dir_result;
+            union {
```

Comment:
I think I would like to have a comment here why we do this.

Reply:
Makes sense, done.